src/main/java/sqltoregex/deparser/SelectDeParserForRegEx.java

```diff
@@ -203,10 +203,19 @@ private String handleAliasAndAggregateFunction(Object o) {
                                                                  o.toString().replaceAll(QUOTATION_MARK_REGEX, "").replaceAll("\\(.*", "")));
             temp.append(OPTIONAL_WHITE_SPACE + "\\(" + OPTIONAL_WHITE_SPACE);
             temp.append(QUOTATION_MARK_REGEX_ZERO_ONE);
-            temp.append(SpellingMistake.useOrDefault(
-                    this.columnNameSpellingMistake,
-                    o.toString().replaceAll(QUOTATION_MARK_REGEX, "").split("\\(")[1].split("\\)")[0])
-            );
+
+            String aggregateFunctionInput = o.toString()
+                                            .replaceAll(QUOTATION_MARK_REGEX, "")
+                                            .split("\\(")[1]
+                                            .split("\\)")[0];
+            if (aggregateFunctionInput.equals("*")){
+                temp.append("(?:\\*|.*)");
```

Comment:
Doesn't .* allow anything in the aggregate function parenthesis?

So it would be possible to insert any (even wrong) expression in it

Reply:
done